### PR TITLE
LPS-67863

### PR DESCRIPTION
**Pull request by Ithildir**
@codyhoag: more changelog, thank you! :blush:

### CODE REVIEW

File: modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown [hunk 1]
```diff
@@ -127,6 +127,20 @@ modules with unpublished dependencies.
 only the version overrides that differ from the versions specified in the
 `bnd.bnd` and `packageinfo` files.
 
+## 1.2.5 - 2016-09-01
+
+### Fixed
+- [LPS-67863]: Fixed exception thrown while running
+`gradlew baseline -PsyncRelease` and the
+`.version-override-${project.name}.properties` file does not exist.
+
```
Comment (codyhoag) [inline modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown]: @Ithildir Would it still be correct if I rephrased this as the following?:

> Avoid throwing an exception when running `gradlew baseline -PsyncRelease` on a project that does not contain a `.version-override-${project.name}.properties` file.

Comment (Ithildir) [inline modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown]: Of course, and it'd sound way way better! Thank you!

**Comment by brianchandotcom** [comment]
Merged. Thx.

**Comment by brianchandotcom** [comment]
@codyhoag please send me a pull? Changes are in upstream. Thx.

**Pull request closed without merging**